hal_st/synchronous_stm32fxxx/SynchronousAesStm.cpp

```diff
@@ -0,0 +1,133 @@
+#include "hal_st/synchronous_stm32fxxx/SynchronousAesStm.hpp"
+#include "infra/util/MemoryRange.hpp"
+#include "infra/util/ReallyAssert.hpp"
+
+#if defined(HAS_PERIPHERAL_AES)
+
+namespace hal
+{
+    SynchronousAesEcbStm::SynchronousAesEcbStm(const Config& config)
+        : config(config)
+    {}
+
+    SynchronousAesEcbStm::~SynchronousAesEcbStm()
+    {
+        Disable();
```

Comment:
Why is the Disable in the destructor, and the Enable in Encrypt/Decrypt? Can't there be symmetry? I'd expect the constructor to contain the Enable